docs/content/getting-started/create-new-project.mdx

```diff
@@ -5,79 +5,117 @@ description: Dagster comes with a convenient CLI command for creating a new proj
+To get started, you can run:
 
 ```bash
-pip install --editable .
+pip install dagster
+dagster project from-example \
+  --name my-dagster-project \
+  --example project_fully_featured
 ```
 
-2. Start the Dagit process. This will start a Dagit web server that, by default, is served on <http://localhost:3000>.
+The command `dagster project from-example` downloads one of the official Dagster examples to the current directory. This command enables you to quickly bootstrap your project with an officially maintained example.
+
+For more info about the examples, visit the [Dagster GitHub repository](https://github.com/dagster-io/dagster/tree/master/examples) or use `dagster project list-examples`.
+
+## Getting started
+
+The newly generated `my-dagster-project` directory is a fully functioning [Python package](https://docs.python.org/3/tutorial/modules.html#packages) and can be installed with `pip`. To install it as a package and its Python dependencies, run:
+
+```bash
+pip install -e .
+```
+
+<Note>
+  By using the <code>--editable</code> flag, <code>pip</code> will install your
+  repository in{" "}
+  <a href="https://pip.pypa.io/en/latest/reference/pip_install/?highlight=editable#editable-installsl">
+    "editable mode"
+  </a>{" "}
+  so that as you develop, local code changes will automatically apply.
+</Note>
+
+Then, start the Dagit web server:
 
 ```bash
 dagit
 ```
 
-The Dagit process automatically uses the file `workspace.yaml` to find your repositories, from which Dagster will load your jobs, schedules, and sensors. To see how you can customize the Dagit process, run `dagit --help`.
+Open `http://localhost:3000` with your browser to see the project.
 
-3. (Optional) If you want to enable Dagster [Schedules](/concepts/partitions-schedules-sensors/schedules) or [Sensors](/concepts/partitions-schedules-sensors/sensors) for your jobs, start the [Dagster Daemon process](/deployment/dagster-daemon) **in the same folder as your `workspace.yaml` file, but in a different shell or terminal**. The `$DAGSTER_HOME` environment variable must be set to a directory for the daemon to work. Note: using directories within `/tmp` may cause issues. See <https://docs.dagster.io/deployment/dagster-instance#default-local-behavior> for more details.
+Now, you can start writing assets in `my_dagster_project/assets/`, define your own ops or jobs and include them in `my_dagster_project/repository.py/`.
+
+## Development
+
+### Adding new Python dependencies
+
+You can specify new Python dependencies in `setup.py`.
+
+### Unit testing
+
+Tests can be added in the `my_dagster_project_tests` directory and you can run tests using `pytest`:
 
 ```bash
-dagster-daemon run
+pytest my_dagster_project_tests_tests
 ```
 
-Once your Dagster Daemon process is running, you should be able to enable schedules and sensors for your Dagster jobs.
+### Schedules and sensors
```

Comment:
do we want to recommend where users put the code for their schedules/sensors? like recommend putting them in repository.py when they are first getting started, and then refactor when they start writing more schedules?